app/templates/events/view/sessions.hbs

```diff
@@ -1,6 +1,6 @@
 {{#if model.isSessionsSpeakersEnabled}}
   <div class="ui grid stackable">
-    {{#if (not-eq session.currentRouteName 'events.view.sessions.create')}}
+    {{#if (and (not-eq session.currentRouteName 'events.view.sessions.create') (not-eq session.currentRouteName 'events.view.sessions.edit'))}}
```

Comment:
Try to reduce the complexity of conditionals in templates. handlebars was designed to keep as less of logic in the template as possible. Move this to a computed property on the controller for better readability.